packages/chart/docs/WordCloud.md

```diff
@@ -0,0 +1,115 @@
+# WordCloud
+
+```meta
+{
+    "source": "https://github.com/hpcc-systems/Visualization/blob/master/packages/chart/src/WordCloud.ts#L11",
+    "extends": "SVGWidget"
+}
+```
+
+A word cloud chart displays words with prominence relative to the word’s given weight. The larger the weight value, relative to the other word weights, the bigger and bolder it appears in the word cloud.
+```sample-code
+import { WordCloud } from "@hpcc-js/chart";
+
+new WordCloud()
+    .columns(["Category", "Value"])
+    .data([
+        ["Apples", 34],
+        ["Bananas", 55],
+        ["Carrots", 89],
+        ["Danishes", 144],
+        ["Eggs", 60],
+        ["Figs", 72],
+        ["Ginger", 92],
+        ["Hazelnut", 102],
+        ["Incaberries", 52],
+        ["Jambalaya", 42],
+    ])
+    .target("target")
+    .render()
+    ;
+```
+Use _fontFamily_ to set the font family. _fontSizeFrom_ and _fontSizeTo_ can be used to control the range of font sizes from the minimum word weight to the maximum word weight.
```

Comment:
Blank line before paragraph (to see why this might be need, put two paragraphs next to each other without the blank line and you will see they run into each other!

Reply:
@jbrundage bump